supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/MachineConstraints.scala

```diff
@@ -21,6 +23,13 @@ object MachineConstraints {
   private val maxMemoryPerCpu = MemorySize(6.5, MemoryUnit.GB)
   private val memoryFactor = MemorySize(256, MemoryUnit.MB)
 
+  private lazy val log = LoggerFactory.getLogger("MachineConstraints")
+  private lazy val usePredefinedMachineTypes: Boolean = {
+    val predefinedMachineTypes = ConfigFactory.load().getOrElse("google.papiv2-use-v1-style-machine-types", false)
+    log.info("PAPI v2 alpha 1 using PAPI v1-style predefined machine types? " + predefinedMachineTypes)
```

Comment:
Also probably not quite ready for production (spamming a non-job-specific log with the exact same message every time a machine config is requested)

Reply:
Oh wait, this is in a lazy val in an object. So should be a one-off. Ok.